pyproject.toml

```
@@ -33,23 +33,22 @@ exclude = [
 "Bug Tracker" = "https://github.com/python-poetry/poetry/issues"
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.7"
 
 # required for compatibility
 importlib-metadata = {version = ">=1.7.0", python = "<3.8"}
-dataclasses = {version = ">=0.8", python = "~3.6"}
 
 [tool.poetry.dev-dependencies]
-pre-commit = {version = "^2.15.0", python = "^3.6.1"}
+pre-commit = {version = "^2.15.0", python = "^3.7.0"}
 pyrsistent = "^0.18.0"
 pytest = "^6.2"
 pytest-cov = "^2.8"
 pytest-mock = "^3.5"
 tox = "^3.0"
 vendoring = {version = "^0.3", python = "^3.8"}
 pep517 = "^0.11.0"
-black = {version = "^21.9b0", markers = "python_version >= '3.6.2' and python_version < '4.0' and implementation_name != 'pypy'"}
-isort = {version = "^5.9.3", python = "^3.6.1"}
+black = {version = "^21.9b0", markers = "python_version >= '3.7.0' and python_version < '4.0' and implementation_name != 'pypy'"}
+isort = {version = "^5.9.3"}
```

just to be consistent:
```suggestion
isort = "^5.9.3"
```

---

Another good catch